pkg/deployments/src/actionId.ts

```diff
@@ -8,7 +8,7 @@ import Task, { TaskMode } from './task';
 
 const ACTION_ID_DIRECTORY = path.join(__dirname, '../action-ids');
 
-type ContractActionIdData = { useAdaptor: boolean; exampleContractAddress?: string; actionIds: Record<string, string> };
+type ContractActionIdData = { useAdaptor: boolean; factoryImplementation?: string; actionIds: Record<string, string> };
```

Comment:
tbh i'm not a fan of this name

Reply:
`factoryProduct` seems ok for now but maybe `factoryOutput`?

Reply:
I like it.